### PR TITLE
Summit

### PR DESCRIPTION
**Pull request by NataliaPoletaeva**
I am glad to share the capstone project I have created in the past week! 

In this project I have:
- created the mark-up for two pages;
- created the styling for two pages;
- adapted two pages for mobile and desktop versions;
- created a hamburger menu  in the mobile version and added functionality to it;
- made the "Speakers" section created dynamically.

Disclaimer: not all the swatch colors actually match the color in the design itself. I have implemented the requested changes using a color-picker, as the design is controversial to itself at some points. 

You can watch my presentation video about the project [here](https://www.loom.com/share/da0550bee7d94d10931e0851f3c25f8f).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+                <li class="element-card">
+                    <img src="https://img.icons8.com/external-inipagistudio-mixed-inipagistudio/64/ffffff/external-strategy-business-mentoring-inipagistudio-mixed-inipagistudio.png"
+                        alt="projects" />
+                    <span class="element-title">Projects</span>
+                    <p class="element-brief">Figure out the new way of solving ecological problems and plan out the
+                        events to make it come true</p>
+                </li>
+            </ul>
+            <button class="join" type="submit">Join EP Global Summit 2021</button>
+            <div class="see-program">SEE THE WHOLE PROGRAM</div>
```
Comment (DeliceLydia) [inline index.html:156]: - [ ] Please underline this title as the design
line

File: index.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+            <p class="event-brief-descr">A productive gathering of ecological experts from all around the world with aim
+                to change the situation with plastic pollution. Takes place on November, in France. </p>
+            <div class="event-details">
+                <div class="event-date">23(TUE) ~ 25.11.2021(THU)</div>
+                <div class="event-place">@ National Museum of Natural History in Paris, la Cité des Sciences et de
+                    l'Industrie and more</div>
+            </div>
+        </section>
+        <section class="event-program">
+            <h2 class="program-head">Main Program</h2>
```
Comment (DeliceLydia) [inline index.html:119]: - [ ] Underline this as well
ye

File: index.html [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+                    <p class="element-brief">Figure out the new way of solving ecological problems and plan out the
+                        events to make it come true</p>
+                </li>
+            </ul>
+            <button class="join" type="submit">Join EP Global Summit 2021</button>
+            <div class="see-program">SEE THE WHOLE PROGRAM</div>
+        </section>
+        <section class="speakers">
+            <h2 class="speakers-head">Featured Speakers</h2>
+        </section>
```
Comment (DeliceLydia) [inline index.html]: - [ ] Use the same colors and fonts like the design
- Basic color swatch: `#272A31`, `#EC5242`, `#D3D3D3`
- Fonts used originally: Cocogoose, Lato

Design
speakers

your page
spef

File: index.html [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+            </ul>
+        </div>
+        <div class="host">
+            <div class="host-logo">
+                <img src="https://img.icons8.com/external-tulpahn-outline-color-tulpahn/64/ffffff/external-ecological-ecology-tulpahn-outline-color-tulpahn.png"
+                    alt="ECOPLANET">
+                <div class="host-info">EcoPlanet</div>
+            </div>
+            <p class="host-descr">2021 EcoPlanet France. All rights reserved.</p>
+        </div>
```
Comment (DeliceLydia) [inline index.html]: - [ ] Please use same background color on this section

your page
prt

original
partner

File: index.html [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+            <button class="join" type="submit">Join EP Global Summit 2021</button>
+            <div class="see-program">SEE THE WHOLE PROGRAM</div>
+        </section>
+        <section class="speakers">
+            <h2 class="speakers-head">Featured Speakers</h2>
+        </section>
+    </main>
+    <footer>
+        <div class="partners">
+            <h2 class="partner-head">Partners</h2>
```
Comment (DeliceLydia) [inline index.html:165]: - [ ] Underline this header

File: index.html [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+                    <span class="element-title">Projects</span>
+                    <p class="element-brief">Figure out the new way of solving ecological problems and plan out the
+                        events to make it come true</p>
+                </li>
+            </ul>
+            <button class="join" type="submit">Join EP Global Summit 2021</button>
+            <div class="see-program">SEE THE WHOLE PROGRAM</div>
+        </section>
+        <section class="speakers">
+            <h2 class="speakers-head">Featured Speakers</h2>
```
Comment (DeliceLydia) [inline index.html:159]: - [ ] Actually you should underline all `hx` check the design

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @NataliaPoletaeva, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- I suggest that you improve how you write your PR you can refer to this
PR




### Keep Rocking!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (NataliaPoletaeva) [reply index.html:156]: Actually, it underlines on hover, just like the border in the program section appears on hover.

**Review by VanessaAoki — APPROVED** [review]
## STATUS: APPROVED 🟢 
Hi @NataliaPoletaeva , 

Great job implementing all the changes required by the previous Code Reviewer, your project is complete! There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

approved

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 
- [x] Awesome design and choice of topic 💯 
- [x] Great video presentation 🎥   
- [x] Correct use of licenses (both MIT and the Creative Commons license) 🔝  


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**